mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
@wmvanvliet you're telling me that we should not allow to compute bipolar ref on EEG with avg ref on? Currently it's possible but the meta data is then not correct...

@jona-sassenhagen you're saying that you don't want the remaining EEG channels (not part of the bipolar montage) to be removed? If so can you still look at a topography etc?